### DISCUSSION TITLE
Install the software

### DISCUSSION THREAD

**Discussion by MarjanFirouznia** [CemrgAppDevelopers/CemrgApp#84] [Q&A]
Hi 
I tried to install the CemrgApp.exe on windows 11 using binary executable file [CemrgApp-Windows-v2.2.zip]. after installation, I can see a black page. can you help me to solve the problem?

**Comment by alonsoJASL** [comment]
Hi @MarjanFirouznia, in windows, opening the app shows a small black window followed by the normal user interface of CemrgApp. The black window will show the log, but it should not be a problem. Although, it seems you are only seeing the black window. Can you please answer the following so we can pinpoint the problem? 
1. How did you install CemrgApp
2. How are you launching the app, i.e. what is the name of the icon you're clicking? In windows, the folder has a lot of files which could be confusing so if you want to share screenshots that would be helpful.
3. Please share a screenshot of your desktop showing the black window you described. 

Thank you!
